packages/cfd/src/Containers/jurisdiction-modal/jurisdiction-modal.tsx

```diff
@@ -33,6 +33,13 @@ const JurisdictionModal = ({
     has_submitted_cfd_personal_details,
 }: TJurisdictionModalProps) => {
     const [checked, setChecked] = React.useState(false);
+    const [cardFlipStatus, setCardFlipStatus] = React.useState({
+        svg: false,
+        bvi: false,
+        vanuatu: false,
+        labuan: false,
+        maltainvest: false,
+    });
```

Comment:
Should'nt we consider which account type card is flipped?
synthetic_svg or financial_svg?

Reply:
@amina-deriv at a time, when a user is viewing the cards, there will only be one type to be displayed right? either synthetic or financial?

Reply:
In the previous implementation, once a card is flipped, it will be in flipped state when revisited. 
Since we fixed that issue now, it is okay to leave as it is now.